pkg/connector/ble/device_darwin.go

```diff
@@ -1,14 +1,34 @@
 package ble
 
 import (
-	"github.com/go-ble/ble"
-	"github.com/go-ble/ble/darwin"
+	"github.com/teslamotors/vehicle-command/internal/log"
+	"tinygo.org/x/bluetooth"
 )
 
-func newDevice() (ble.Device, error) {
-	device, err := darwin.NewDevice()
-	if err != nil {
-		return nil, err
+func IsAdapterError(err error) bool {
```

Comment:
Function causes `make linters` to fail:

```
pkg/connector/ble/device_darwin.go:8:21: unused-parameter: parameter 'err' seems to be unused, consider removing or renaming it as _ (revive)
```

I.e., change to `IsAdapterError(_ error) bool {`